numpy/core/tests/test_dtype.py

```diff
@@ -126,6 +126,15 @@ def test_numeric_style_types_are_invalid(self, dtype):
         with assert_raises(TypeError):
             np.dtype(dtype)
 
+    def test_remaining_dtypes_with_bad_bytesize(self):
+        # These should probably deprecated as well, the np.<name> aliases were
```

Comment:
Is this what you meant?
```suggestion
        # The np.<name> aliases were deprecated, these probably should be too 
```